docs/userguides/development.md

```diff
@@ -14,6 +14,8 @@ If silverback finds a module named `bot.py` either in the `bots/` directory or a
 It will also be able to detect the scripts inside your `bots/` directory and let you run those by name (in case you have multiple bots in your project).
 It is suggested that you create the instance of your `SilverbackApp()` object by naming the variable `bot`, since `silverback` will autodetect that variable name when loading your script file.
 
+Another way you can structure your bot is to create a `bot` folder and define a runner inside of that folder as `__init__.py`. This method of setting up your project will not allow you to use `silverback build` currently, and you will have to define your own dockerfile. There is a below example of how to run an app defined in this manner.
```

Comment:
I think we can support this with the docker file though, just see if it's a `bot.py` or `bot` folder and copy over appropiately

the third case is that a script is under `bots/{name}.py`, which will get copied in the dockerfile from that location to `bot.py` (the first case which we support automatic loading for)

Reply:
The endgame here is that each of the 3 "suggested" ways of defining a bot ends up with a local `bot` module inside of the dockerfile

Reply:
I'll organize this as a list in here